packages/strapi-plugin-upload/admin/src/components/List/index.js

```diff
@@ -2,70 +2,66 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Checkbox } from '@buffetjs/core';
 import { get } from 'lodash';
-import { createMatrix } from '../../utils';
 import Card from '../Card';
 import CardControlsWrapper from '../CardControlsWrapper';
 import ListWrapper from '../ListWrapper';
+import ListCell from './ListCell';
 
 const List = ({
   clickable,
   data,
   onChange,
   onCardClick,
   selectedItems,
+  small,
```

Comment:
I thought this `small` props was used to determine the size of the list. Maybe you can rename to smallCards (or something else) as it is used to set small cards?